esphome/components/pylontech/__init__.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import esphome.codegen as cg
+from esphome.components import uart
+import esphome.config_validation as cv
+from esphome.const import CONF_ID
+
+CODEOWNERS = ["@functionpointer"]
+DEPENDENCIES = ["uart"]
+AUTO_LOAD = ["sensor"]
+_LOGGER = logging.getLogger(__name__)
+MULTI_CONF = True
```

Comment:
Just a small tidy up and removal of auto load that is not needed
```suggestion
_LOGGER = logging.getLogger(__name__)

CODEOWNERS = ["@functionpointer"]
DEPENDENCIES = ["uart"]
MULTI_CONF = True
```